var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -343,19 +350,19 @@ def cmake_args(self):
 
         # 64 bit indices
         options.append(self.define_from_variant(
-            'DEAL_II_WITH_64BIT_INDICES', 'int64')
-        )
+            'DEAL_II_WITH_64BIT_INDICES', 'int64'
+        ))
 
         if (spec.satisfies('^openblas+ilp64') or
             spec.satisfies('^intel-mkl+ilp64') or
             spec.satisfies('^intel-parallel-studio+mkl+ilp64')):
-            options.append('-DLAPACK_WITH_64BIT_BLAS_INDICES=ON')
+            self.define('LAPACK_WITH_64BIT_BLAS_INDICES', True)
```

Comment:
`options.append` is missing here: the line is a no-op.

Reply:
Thanks for spotting this error as well. I've fixed it up!